canonical/canonical-design
# WD-16709 Add GTM & Cookie Policy

## Done
- Added Google Tag Manager
- Added Cookie Consent Banner
- Added Webpack to compile JS
- Minor changes
## QA
- See if you get the banner for cookies.
- Try to revoke cookies from footer
- See if GTM is being connected and scripts being fetched
- See if wepage is using compiled and versioned JS scripts
## Fixes

[WD-16709](https://warthogs.atlassian.net/browse/WD-16709)

[WD-16709]: https://warthogs.atlassian.net/browse/WD-16709?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
